packages/core/strapi/lib/services/entity-service/types/index.d.ts

```diff
@@ -11,4 +11,13 @@ export interface EntityService {
     entityId: number,
     params?: EntityService.Params.Pick<TContentTypeUID, 'fields' | 'populate'>
   ): Promise<Attribute.GetValues<TContentTypeUID> | null>;
+  delete<TContentTypeUID extends Common.UID.ContentType>(
+    uid: TContentTypeUID,
+    entityId: number,
```

Comment:
This might be a string too 🤔

Reply:
I've created this https://github.com/strapi/strapi/blob/c41c3e715d9316fbf2d9179b8167cc845b7b028b/packages/core/strapi/lib/services/entity-service/types/params/attributes.d.ts#L21 ID type for the entity service but we might need to move it somewhere else